data/variables.yml

```diff
@@ -0,0 +1,598 @@
+age_and_sex:
+  verbose_name: Age and Sex
```

Comment:
None of the verbose names in this file are currently being used, but I assume at some point we'll want to use them to create a crosswalk for user display.

Reply:
what is yaml giving us that just having this data as python dict is not?

Reply:
Two primary considerations:

1. YAML is easier to write than Python dicts or JSON, and I had to write this out by hand
2. There's enough data here that it needs to be in its own separate file to avoid cluttering up the code, so we might as well store it in a language-agnostic data format

Reply:
okay